src/nav/root.yml

```diff
@@ -55,6 +55,8 @@ pages:
     path: integrations
   - title: Vulnerability Management
     path: vuln-management
+  - title: Interactive applications security testing (IAST)
```

Comment:
Please move this so it's alphabetical in the nav, between Errors inbox and Open-source integrations

Reply:
Yep!